model/nextcloud/nextcloud.go

```diff
@@ -99,48 +100,54 @@ func New(inst *instance.Instance, accountID string) (*NextCloud, error) {
 		Host:     u.Host,
 		Username: username,
 		Password: password,
+		BasePath: "/remote.php/dav",
 		Logger:   logger,
 	}
 	nc := &NextCloud{
 		inst:      inst,
 		accountID: accountID,
 		webdav:    webdav,
 	}
-	if err := nc.fillBasePath(&doc); err != nil {
+	if err := nc.fillUserID(&doc); err != nil {
 		return nil, err
 	}
 	return nc, nil
 }
 
 func (nc *NextCloud) Download(path string) (*webdav.Download, error) {
-	return nc.webdav.Get(path)
+	return nc.webdav.Get("/files/" + nc.userID + "/" + path)
```

Comment:
Can `path` ever be `/`?

Reply:
Nope, you can only download a file, and `/` is not a correct path for a file.

Reply:
Sure but this line is repeated throughout the file. My point is what happens if any of these methods is called with `path == "/"`?
Won't there be an error?

Reply:
It's not called with `path == "/"`. Even if it would have been the case, it's not an error, it just makes the URL looks worse, but it still works.